.golangci.yml

```diff
@@ -0,0 +1,136 @@
+# https://habr.com/company/roistat/blog/413175/
```

Comment:
We have some nice settings in gate repos, why don't we reuse them here?

Reply:
Sorry, I took this from neofs-node. 
Replaced.